File-Storage-Security/Deployment/aws-nested-aio-promote-quarantine-plugin/aws-aio-nested-w-plugin.yaml

```diff
@@ -0,0 +1,465 @@
+        PostScanActionTagDLQARN: !Ref PostScanActionTagDLQARN
+        ReportObjectKey: !Ref ReportObjectKey
+        S3BucketToScan: !Ref S3BucketToScan
+        ScannerAWSAccount: !Ref AWS::AccountId
+        ScannerLambdaAliasARN: !GetAtt ScannerStack.Outputs.ScannerLambdaAliasARN
+        ScannerSQSURL: !GetAtt ScannerStack.Outputs.ScannerQueueURL
+        ScanOnGetObject: !Ref ScanOnGetObject
+        ScanResultTopicDLQARN: !Ref ScanResultTopicDLQARN
+        SecurityGroupIDs: !Join [',', !Ref SecurityGroupIDs]
+        SNSTopicPrefix: !Ref SNSTopicPrefix
+        SubnetIDs: !Join [',', !Ref SubnetIDs]
+        TrendMicroManagementAccount: !Ref TrendMicroManagementAccount
+        TriggerWithObjectCreatedEvent: !Ref TriggerWithObjectCreatedEvent
+      Tags:
+        - Key: Name
+          Value: FSS-Storage-Stack
+      TemplateURL:
+        Fn::Sub: https://${FSSBucketName}-${AWS::Region}.s3.${AWS::Region}.amazonaws.com/${FSSKeyPrefix}templates/FSS-Storage-Stack.template
+      TimeoutInMinutes: 30
+
+  ScannerStack:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Parameters:
+        AdditionalIAMPolicies: !Join [',', !Ref AdditionalIAMPolicies]
+        CloudOneRegion: !Ref CloudOneRegion
+        ExternalID: !Ref ExternalID
+        FSSBucketName: !Ref FSSBucketName
+        FSSKeyPrefix: !Ref FSSKeyPrefix
+        IAMPolicyPrefix: !Ref IAMPolicyPrefix
+        IAMRolePrefix: !Ref IAMRolePrefix
+        KMSKeyARNForQueueSSE: !Ref KMSKeyARNForQueueSSE
+        KMSKeyARNsForTopicSSE: !Ref KMSKeyARNForTopicSSE
+        LambdaFunctionPrefix: !Ref LambdaFunctionPrefix
+        LambdaLayerPrefix: !Ref LambdaLayerPrefix
+        NetworkProxy: !Ref NetworkProxy
+        PermissionsBoundary: !Ref PermissionsBoundary
+        ScannerEphemeralStorage: !Ref ScannerEphemeralStorage
+        SecurityGroupIDs: !Join [',', !Ref SecurityGroupIDs]
+        SQSQueuePrefix: !Ref SQSQueuePrefix
+        SubnetIDs: !Join [',', !Ref SubnetIDs]
+        TrendMicroManagementAccount: !Ref TrendMicroManagementAccount
+      Tags:
+        - Key: Name
+          Value: FSS-Scanner-Stack
+      TemplateURL:
+        Fn::Sub: https://${FSSBucketName}-${AWS::Region}.s3.${AWS::Region}.amazonaws.com/${FSSKeyPrefix}templates/FSS-Scanner-Stack.template
+      TimeoutInMinutes: 30
+
+  PluginStack:
+    Type: AWS::CloudFormation::Stack
+    DependsOn: StorageStack
+    Properties:
+      Parameters:
+        ScanningBucketName: !Ref S3BucketToScan
+        ScanResultTopicARN: 
+          Fn::GetAtt:
+          - StorageStack
+          - Outputs.ScanResultTopicARN
+        PromoteBucketName: !Ref PromoteBucketName
+        PromoteMode: !Ref PromoteMode 
+        QuarantineBucketName: !Ref QuarantineBucketName
+        QuarantineMode: !Ref QuarantineMode
+        ACL: !Ref ACL
+      Tags:
+        - Key: Name
+          Value: FSS-Scanner-Stack
+      TemplateURL: https://aws-workshop-c1as-cft-templates.s3.amazonaws.com/p-q.yaml
```

Comment:
How fast/frequent can we update this in case a bug needs to fix or a feature needs to be added?

I was thinking if there is any other method more reliable than this one... 🤔

Reply:
well normally, TM manages the bucket however, I am happy to move to this where ever. I can also attach that cft to this project as well.

Reply:
We can have it "as is" for now, I will create an issue just to keep track of that...